src/support.py

```diff
@@ -19,6 +19,11 @@
 def resource_path(relative_path: str):
     """Get absolute path to resource, works for dev and for PyInstaller"""
     relative_path = relative_path.replace("/", os.sep)
+
+    # Needed for pygbag runtime environment compatibility:
+    if sys.platform in ("emscripten", "wasm"):
+        return relative_path
+
```

Comment:
Actually, would the async usage even impact normal execution (i.e. not in pygbag)?